<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auth</class>
 <widget class="QDialog" name="Auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auth</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/img/img/focus.png</normaloff>:/img/img/focus.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="6" column="1">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QFrame" name="frame_3">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="1" column="1">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="13" column="1">
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="12" column="1">
                 <widget class="QPushButton" name="registerLinkBtn">
                  <property name="font">
                   <font>
                    <pointsize>8</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>New user? Click here to create a new account</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="7" column="2">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>260</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="label_10">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Login</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>259</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="7" column="1">
                 <layout class="QFormLayout" name="formLayout">
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <item row="3" column="0" colspan="2">
                   <widget class="QLineEdit" name="passwordLoginEdit">
                    <property name="inputMethodHints">
                     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Password</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <widget class="QLineEdit" name="loginEdit">
                    <property name="inputMethodHints">
                     <set>Qt::ImhEmailCharactersOnly</set>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" colspan="2">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Username/Email</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="8" column="1">
                 <widget class="QPushButton" name="loginBtn">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">padding: 12px 20px;
border: 1px solid;
border-radius: 5px;
background-color: rgb(52, 101, 164);
color: white;</string>
                  </property>
                  <property name="text">
                   <string>Login</string>
                  </property>
                  <property name="autoDefault">
                   <bool>true</bool>
                  </property>
                  <property name="default">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QFrame" name="frame_4">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="17" column="1">
                 <widget class="QPushButton" name="registerBtn">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                    <kerning>true</kerning>
                   </font>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">padding: 12px 20px;
border: 1px solid;
border-radius: 5px;
background-color: rgb(52, 101, 164);
color: white;</string>
                  </property>
                  <property name="text">
                   <string>Register</string>
                  </property>
                  <property name="autoDefault">
                   <bool>true</bool>
                  </property>
                  <property name="default">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="label_11">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Register</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="19" column="1">
                 <spacer name="verticalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="5" column="0">
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>259</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="5" column="2">
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>260</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="1">
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="18" column="1">
                 <widget class="QPushButton" name="loginLinkBtn">
                  <property name="font">
                   <font>
                    <pointsize>8</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Already have an account? Click here to login</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <layout class="QFormLayout" name="formLayout_2">
                  <property name="topMargin">
                   <number>10</number>
                  </property>
                  <item row="0" column="0" colspan="2">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>First Name</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <widget class="QLineEdit" name="firstNameEdit">
                    <property name="cursorPosition">
                     <number>0</number>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Last Name</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0" colspan="2">
                   <widget class="QLineEdit" name="lastNameEdit"/>
                  </item>
                  <item row="4" column="0" colspan="2">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Username</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="0" colspan="2">
                   <widget class="QLineEdit" name="usernameEdit">
                    <property name="inputMethodHints">
                     <set>Qt::ImhPreferLowercase</set>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="0" colspan="2">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Email</string>
                    </property>
                   </widget>
                  </item>
                  <item row="7" column="0" colspan="2">
                   <widget class="QLineEdit" name="emailEdit">
                    <property name="inputMethodHints">
                     <set>Qt::ImhEmailCharactersOnly</set>
                    </property>
                   </widget>
                  </item>
                  <item row="8" column="0" colspan="2">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>Password</string>
                    </property>
                   </widget>
                  </item>
                  <item row="9" column="0" colspan="2">
                   <widget class="QLineEdit" name="passwordEdit">
                    <property name="inputMethodHints">
                     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="10" column="0" colspan="2">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Confirm Password</string>
                    </property>
                   </widget>
                  </item>
                  <item row="11" column="0" colspan="2">
                   <widget class="QLineEdit" name="confirmEdit">
                    <property name="inputMethodHints">
                     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
                    </property>
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <stylestrategy>PreferDefault</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Focus</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>loginEdit</tabstop>
  <tabstop>passwordLoginEdit</tabstop>
  <tabstop>loginBtn</tabstop>
  <tabstop>registerLinkBtn</tabstop>
  <tabstop>firstNameEdit</tabstop>
  <tabstop>lastNameEdit</tabstop>
  <tabstop>usernameEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>confirmEdit</tabstop>
  <tabstop>registerBtn</tabstop>
  <tabstop>loginLinkBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
